README.md

```diff
@@ -1,40 +1,82 @@
+- `pip install -e .` installs the package in the virtual environment
+
+## System setup
+Please refer to [this SETUP guide](SETUP.md) which guides you through all the steps to setup tools needed like:
+- setting up s3 emulation servers through minio
+- setting up transfer tools like Rclone, MFT, Apache Nifi
 
-### Apache Nifi
+# Configuration Setup
+We use YAML-based configuration to feed configuration to our pipeline.
+Edit the configuration file to update the S3 buckets details `data-transfer-evaluation/tests/config.yaml`.
 
-* Download and unarchive Nifi binary from https://dlcdn.apache.org/nifi/1.15.3/nifi-1.15.3-bin.tar.gz
-* Update property nifi.security.allow.anonymous.authentication=true in conf/nifi.properties
-* Start Nifi server ```./bin/nifi.sh start --wait-for-init 120```
-* After tests are completed, run ```./bin/nifi.sh start``` to stop the server
+## Transfer configuration YAML sample
 
-### Rclone
+```YAML
+---
+source_token: <username>
+source_secret: <password>
+source_s3_endpoint: "http://127.0.0.1:8080"
+source_s3_bucket: "src"
+source_s3_region: "us-east-1"
+
+dest_token: <username>
+dest_secret: <password>
+dest_s3_endpoint: "http://127.0.0.1:8090"
+dest_s3_bucket: "dest"
+dest_s3_region: "us-east-1"
+```
 
-* Download and install rclone binary from ```curl https://rclone.org/install.sh | sudo bash```
+### For Source Server:
+* `source_token` represents minio username
+* `source_secret` represents minio password
+* `source_s3_endpoint` is where we are running the minio source server
+* `source_s3_bucket` is the name of the source directory
+* `source_s3_region` emulates s3 region (just use defaults)
 
-### Airavata MFT
+### For Destination Server:
+* `dest_token` represents minio username
+* `dest_secret` represents minio password
+* `dest_s3_endpoint` is where we are running the minio destination server
+* `dest_s3_bucket` is the name of the destination directory
+* `dest_s3_region` emulates s3 region (just use defaults)
 
-* Download and unarchive MFT Binary from https://github.com/apache/airavata-mft/releases/download/0.1-pre-release/airavata-mft-0.1.zip
-* Start consul server by running ```./start_consul.sh mac``` or ```./start_consul.sh linux``` depending on the operating system
-* Start Airavata MFT by running ```./start_mft.sh```
-* After tests are completed, run ```./stop_mft.sh && ./stop_consul.sh``` to stop MFT services
+# Test Run Eval Framework
 
-### Examples and usages
+* Make sure minio servers and transfer tools are up and configured (see [setup guide](SETUP.md))
+* Activate virtual environment if you have it set
+* Create a transfer configuration YAML file (see the above "Configuration Setup" section for reference)
+* Make sure you have s3 src/dest buckets (see [setup guide](SETUP.md) for details; or just run `scripts/buckets.sh` script)
+* Make sure minio servers are up and running (see [setup guide](SETUP.md))
+* Before running this, you need to export few environment variables
```

Comment:
Do we need `Before running this`?

Reply:
not required though as the user will follow the steps provided in the README.md file